alacritty/src/display/content.rs

```diff
@@ -307,8 +307,12 @@ impl RenderableCell {
         let config = &content.config;
         match fg {
             Color::Spec(rgb) => match flags & Flags::DIM {
-                Flags::DIM => rgb * DIM_FACTOR,
-                _ => rgb,
+                Flags::DIM => {
+                    // FIXME: ???
```

Comment:
???

Reply:
Fixed in https://github.com/alacritty/alacritty/pull/6896/commits/d067434b45ddb26b5dd181a98d6fc022962a9a51.